src/games/strategy/util/SHA256Crypt.java

```diff
@@ -0,0 +1,23 @@
+package games.strategy.util;
+
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * Replacement for MD5Crypt
+ * using  SHA256 instead
+ */
+public class SHA256Crypt {
+  /**
+   * Returns the SHA256-Hash of the given String
+   */
+  public static String crypt(String text) throws UnsupportedEncodingException, NoSuchAlgorithmException{
+    MessageDigest md = MessageDigest.getInstance("SHA-256");
+
+    md.update(text.getBytes("UTF-8"));
```

Comment:
java nio has the constant for UTF-8: https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html